react/week3/todolist/src/App.css

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Please remove CSS styles that you don't use

Reply:
Its just notes of the font and the color palette that I used so its easier to find if I continue adding styles.